WalletWasabi/Blockchain/TransactionOutputs/CoinsRegistry.cs

```diff
@@ -293,4 +258,36 @@ public ICoinsView AsAllCoinsView()
 	public ICoinsView Unspent() => AsCoinsView().Unspent();
 
 	IEnumerator IEnumerable.GetEnumerator() => AsCoinsView().GetEnumerator();
+
+	public bool Seen(uint256 txid)
+	{
+		lock (Lock)
+		{
+			return Transactions.ContainsKey(txid);
+		}
+	}
+
+	public SmartCoin[] GetMyInputs(SmartTransaction transaction)
+	{
+		var inputs = transaction.Transaction.Inputs.Select(x => x.PrevOut).ToArray();
+
+		var myInputs = new List<SmartCoin>();
```

Comment:
No for a handful of reasons, but mainly because `WalletInputs` is common to all opened wallets.
Don't worry I made the same mistake in my `TransactionSummary` PR